lib/Controller/MessagesController.php

```diff
@@ -908,7 +908,7 @@ public function smartReply(int $messageId):JSONResponse {
 			return new JSONResponse([], Http::STATUS_FORBIDDEN);
 		}
 		try {
-			$replies = $this->aiIntegrationService->getSmartReply($account, $mailbox, $message, $this->currentUserId);
+			$replies = array_values($this->aiIntegrationService->getSmartReply($account, $mailbox, $message, $this->currentUserId));
```

Comment:
Breaks if the method returns null when the reply of the LLM could not be parsed. Need to check if the result of `getSmartReply()` is null before passing it to `array_values()`.

`array_values(): Argument #1 ($array) must be of type array, null given in file '/home/richard/src/nextcloud/master/dev_apps/mail/lib/Controller/MessagesController.php' line 911`